inc/api.php

```diff
@@ -89,6 +89,87 @@ function prepare_post_items( array $items ) : array {
 	return $posts;
 }
 
+/**
+ * Get the read more link text, based on the post type and source.
+ *
+ * @param string $external_src_link   The URL of the original post source (photo, video, linked article).
+ * @param string $social_post_src     The social network the post is pulled from.
+ * @param string $social_profile_name The profile name of the social network account.
+ *
+ * @return string                     The filtered Juicer post message.
+ */
+function get_read_more_text( $external_src_link, $social_post_src, $social_profile_name ) {
+	// Get just the host portion of the URL.
+	$external_src_href = wp_parse_url( $external_src_link )['host'];
+	$external_src_href = str_replace( 'www.', '', $external_src_href );
+
+	// If the JUICER_SHORT_URL constant is set, get the value.
+	$short_url = get_short_url();
+	if ( $short_url ) {
+		// Get just the host portion of the URL.
+		$short_url = wp_parse_url( $short_url )['host'];
+		$short_url = str_replace( 'www.', '', $short_url );
+	}
+
+	// If the JUICER_LONG_URL constant is set, get the value.
+	$long_url = get_long_url();
+	if ( $long_url ) {
+		// Get just the host portion of the URL.
+		$long_url = wp_parse_url( $long_url )['host'];
+		$long_url = str_replace( 'www.', '', $long_url );
+	}
+
+	// If the JUICER_SITE_NAME constant is set, get the value.
+	if ( get_site_name() ) {
+		$site_name = get_site_name();
+	} else {
+		$site_name = $social_profile_name;
+	}
+
+	/**
+	 * If the original linked article is from one of the URLs provided in the constants,
+	 * use the site name constant or social network profile name.
+	 *
+	 * If the url contains the social network name and the keyword video or photo,
+	 * set the post type acordingly and use the name of the social network.
+	 *
+	 * Otherwise, use the host portion of the external_src_link url.
+	 */
+	if ( $short_url === $external_src_href
+		|| $long_url === $external_src_href ) {
+		$post_type     = 'post';
+		$read_more_src = $site_name;
+
+	} elseif ( strpos( $external_src_link, strtolower( $social_post_src ) ) && strpos( $external_src_link, 'video' ) ) {
+		$post_type     = 'video';
+		$read_more_src = $social_post_src;
+
+	} elseif ( strpos( $external_src_link, strtolower( $social_post_src ) ) && preg_match( '(image|photo)', $external_src_link ) ) {
+		$post_type     = 'photo';
+		$read_more_src = $social_post_src;
+
+	} else {
+		$post_type     = 'post';
+		$read_more_src = $external_src_href;
+	}
+
+	// Set text based on post type.
+	if ( 'photo' === $post_type ) {
+		// translators: the item source (Facebook, Twitter, Original Website).
```

Comment:
Translator comments should be capitalized (like any other comment) and should note what the comment is referring to (in this case the `%s`. The idea here is that a translator will be using a program to translate strings and might need context to understand what the variable in the string is referring to to provide an accurate translation.
🔢 

```suggestion
		// Translators: %s refers to the item source (Facebook, Twitter, Original Website).
```

Reply:
Fixed in [1021254](https://github.com/humanmade/hm-juicer/pull/6/commits/1021254e6a720b596ee55f1fdb7b0dde2f580ca7)